content/doc/developer/security/secrets.adoc

```diff
@@ -80,3 +80,11 @@ Secrets spanning multiple lines (such as certificates or SSH keys) should be mas
 It is available from Jenkins 2.171.
 Plugins with older core baselines can add a dependency on the standalone library `io.jenkins.temp.jelly:multiline-secrets-ui:1.0` (or newer) with the same form element.
 See https://github.com/jenkinsci/lib-multiline-secrets-ui/blob/master/README.md[the documentation in its GitHub repository] for usage instructions and sample code.
+
+### Encryption of Secrets and Credentials
+
+Jenkins uses a master key which is a 128-bit AES key used for encrypting and decrypting secrets and credentials.
+This key is store _unencrypted_ on the Jenkins controller filesystem in `$JENKINS_HOME/secrets/master.key` and is used as a key encryption key to protect the individual AES keys used for protecting secrets.
```

Comment:
is it necessary to mention key twice in 3 words? 'key encryption key' it sounds funny is there a different way to phrase it?

Reply:
They're also called "KEKs", but that's also a meme word more than "key encryption key" is. I agree that this could use some revision of the word key, though.

Reply:
I'll note that in a parenthetical while being a little less jargony in the main sentence.